addons/event/data/event_demo.xml

```diff
@@ -14,6 +14,14 @@
         <field name="event_type_id" ref="event_type_0"/>
         <field name="stage_id" ref="event_stage_booked"/>
         <field name="tag_ids" eval="[(4, ref('event.event_tag_category_1_tag_1')), (4, ref('event.event_tag_category_2_tag_1'))]"/>
+        <field name="ticket_instructions" type="html">
```

Comment:
Would add it to our famous event online demo event also :D

Reply:
Nouredine had to remove the "online" word from "online event". We could re-try to do it and add ticket instructions :D #perfectdemodata (ok, don't do it)